Addings details on feature variables from tests (#6409)
DataDog/documentation: content/en/synthetics/settings/_index.md

This feature is in private beta, <a href="/help">contact Datadog support</a> to turn on this feature for your account.
</div>

> You can create variables using parsed response headers and body from your existing HTTP tests. Variables are updated at the frequency of the test it is parsing from. 

Reviewer: ```suggestion
You can create variables from your existing HTTP tests by parsing its response headers or body. Variables are updated with the same frequency as the test it's coming from:
```
